mypy.ini

```diff
@@ -11,7 +11,8 @@ disallow_any_expr=False
 disallow_any_decorated=False
 disallow_any_explicit=False
 disallow_any_generics=True
-disallow_subclassing_any=False  # note: currently only required for templates/scraper.py
+# note: currently only required for templates/scraper.py
+disallow_subclassing_any=False
```

Comment:
Leaving that comment on the same line as the configuration setting is intended to indicate that comment only refers to that single setting (and to make it more difficult for line rearrangements and edits in this file in future to lose that information accidentally).

Reply:
Mypy was complaining about it being an invalid option

I don’t think ini supports comments on the end of lines

Reply:
Ek, hm.  I think you're right - for _some_ implementations of ini-files, at least (including the one currently used by `mypy`).

I tested it out locally, and see the same warning.  So, in summary: it behaves "as we intend" (`disallow_subclassing_any` has a `false` value after the config is parsed) -- but not because of this line in the config.

How difficult would it be to fix the underlying issue so that we can set the value to `true` and move on without needing a comment at all?

Reply:
another option would be to move it to the `project.toml` which has more reasonable comment behavior